### PR TITLE
Updates for Couchbase Server 7.0.0

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

File: aws/couchbase-ami-creation/ami_copy.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+while getopts gr:i: flag
+do
+    case "${flag}" in
+        r) REGION=${OPTARG};;
+        i) AMI_ID=${OPTARG};;
+        g) GATEWAY=1;;
+        *) exit 1;;
+    esac
+done
+TAB="    "
```
Comment (patrick-stephens) [inline aws/couchbase-ami-creation/ami_copy.sh:15]: What's the reason for this over \t?

File: aws/couchbase-ami-creation/rpm_exploder.sh [hunk 2]
```diff
@@ -63,21 +63,29 @@ service disable-thp start
 
 
 # Grab installer in case we need it and the user doesn't use the pre-installed
-wget -O /setup/couchbase_installer.sh __SCRIPT_URL__
+wget -O /setup/couchbase_installer.sh https://github.com/couchbase-partners/marketplace-scripts/releases/download/v1.0.13/couchbase_installer.sh
```
Comment (patrick-stephens) [inline aws/couchbase-ami-creation/rpm_exploder.sh:66]: Should this be parameterized still?

**Review by patrick-stephens — DISMISSED** [review]

File: aws/couchbase-ami-creation/ami_copy.sh (hunk 1 above)
Comment (malscent) [reply aws/couchbase-ami-creation/ami_copy.sh:15]: spaces vs tabs

File: aws/couchbase-ami-creation/rpm_exploder.sh (hunk 2 above)
Comment (malscent) [reply aws/couchbase-ami-creation/rpm_exploder.sh:66]: ahh yeah... i didn't fix that one to not be idempotent

**Review by patrick-stephens — APPROVED** [review]

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**